book/src/config_api.md

```diff
@@ -12,6 +12,8 @@ target_path = "."
 # Path where objects generated (defaults to <target_path>/src/auto)
 # auto_path = "src/auto"
 work_mode = "normal"
+# Whether the library uses https://gitlab.gnome.org/GNOME/gi-docgen for its documentation
+use_gi_docgen = true
```

Comment:
The default is false, no?

Reply:
it's, yes; will update it to use the default value